docs/swaggerDefinition.js

```diff
@@ -425,6 +425,14 @@ const swaggerOptions = {
             }
           }
         },
+        userAvailable: {
+          type: 'object',
+          properties: {
+            userAvailable: {
```

Comment:
re: `isUsernameAvailable`

Reply:
done. Changed the property name only to isUsernameAvailable.